lib/SimpleSAML/XML/Errors.php

```diff
@@ -98,8 +101,8 @@ public static function end() {
 	 *
 	 * This function formats the given LibXMLError object as a string.
 	 *
-	 * @param $error  The LibXMLError which should be formatted.
-	 * @return  A string representing the given LibXMLError.
+	 * @param $error \LibXMLError  The LibXMLError which should be formatted.
```

Comment:
Shouldn't this be the other way around?

```
 * @param \LibXMLError $error ...
```

I see this happened all over the code, so we should fix it.